ui/src/components/Layout/SideNavigation.vue

```diff
@@ -50,13 +62,42 @@
 </template>
 
 <script>
-import { serverBus } from "../../main";
 
 export default {
+  computed:{
+    getChevronIcon()
+    {
+      if(this.mini)
+      {
+        return "mdi-chevron-left"
+      }
+      else
+      {
+        return "mdi-chevron-right"
+      }
+    },
+    isSmallScreenSize(){ 
```

Comment:
This method shouldn't be setting a variable. A computed property should only be returning a value. Also, avoid method names starting with "is" as a computed property. So, smallScreenSize for example.

Reply:
Done